R/pkg/R/DataFrame.R

```diff
@@ -1652,34 +1607,43 @@ setMethod("dapplyCollect",
 
 #' gapply
```

Comment:
Shall we update here too? Seems a bit odd that the short summary says gapply for dapply ones:
![Screen Shot 2020-05-19 at 6 16 03 PM](https://user-images.githubusercontent.com/6477701/82308852-529cba00-99fd-11ea-9321-d770e5a26242.png)
![Screen Shot 2020-05-19 at 6 16 15 PM](https://user-images.githubusercontent.com/6477701/82308860-54667d80-99fd-11ea-9c77-f0dbaff02cab.png)

Reply:
We could change something like "R native function execution". There are a lot of examples like this to group it in `functions.R`. Might be best to follow the way consistently.

Reply:
You can check the documentation build via `cd R && ./create-docs.sh && open pkg/html/index.html`

Reply:
Thanks @HyukjinKwon , i went with `Functions to apply native R code on groups or partitions`. 

Generally I think the Rd titles could go for a fresh pass overall (e.g. `cov` is not very descriptive and applies to a bevy of functions), but I'll leave that for another day.